README.md

```diff
@@ -0,0 +1,93 @@
+# Go Snowflake Driver
+
+<a href="https://github.com/snowflakedb/gosnowflake/actions?query=workflow%3A%22Build+and+Test%22">
+    <img src="https://github.com/snowflakedb/gosnowflake/workflows/Build%20and%20Test/badge.svg?branch=master">
+</a>
+<a href="http://www.apache.org/licenses/LICENSE-2.0.txt">
+    <img src="http://img.shields.io/:license-Apache%202-brightgreen.svg">
+</a>
+<a href="https://goreportcard.com/report/github.com/snowflakedb/gosnowflake">
+    <img src="https://goreportcard.com/badge/github.com/snowflakedb/gosnowflake">
+</a>
+
+This topic provides instructions for installing, running, and modifying the Go Snowflake Driver. The driver supports Go's [database/sql](https://golang.org/pkg/database/sql/) package.
+
+# Prerequisites
+
+The following software packages are required to use the Go Snowflake Driver.
+
+## Go
+
+The latest driver requires the [Go language](https://golang.org/) 1.17 or higher. The supported operating systems are Linux, Mac OS, and Windows, but you may run the driver on other platforms if the Go language works correctly on those platforms.
+
+
+# Installation
+
+Get Gosnowflake source code, if not installed.
+
+```sh
```

Comment:
Are the presence and placement of "sh" deliberate? 

My knee-jerk reaction is that it looks as though we're trying to use /bin/sh to run a "go" command, which seems a bit odd, and also seems inconsistent with the fact that the "sh" is on a different line from the "go". 

Or maybe this is MarkDown notation that I don't recognize (I don't know MarkDown).

I know I'm asking questions, not giving answers.  :-(

Reply:
Those were shell commands, which is why I used "sh". It doesn't really make a difference in this code blocks, though, so I just removed them.